Dockerfile

```diff
@@ -26,6 +26,8 @@ RUN pip install -r /opt/pyspider/requirements.txt
 
 # add all repo
 ADD ./ /opt/pyspider
+# install seadaka requ
+RUN pip install -r /opt/pyspider/requirements-seadaka.txt
```

Comment:
Don't your package in base docker image, you could create another Dockerfile that base on this image while install your additional dependency.

Reply:
Sorry for that, I remove it.